playbooks/roles/ocp-config/README.md

```diff
@@ -25,8 +25,9 @@ Role Variables
 | proxy_url               | no       | ""             | Proxy url eg: http://[user:passwd@]server:port (NA when setup_squid_proxy: true)|
 | no_proxy                | no       | ""             | Comma seperated string of domains/cidr to exclude proxy |
 | enable_local_registry   | no       | false          | Set to true to enable usage of local registry for restricted network install |
-| chronyconfig.enabled    | no       | flase          | Set to true to enable chrony configuration on the coreOS node during install |
-| chronyconfig.content    | no       | ""             | List of time servers and options pair (see chronyconfig examples) |
+| chronyconfig.enabled    | no       | true           | Set to true to enable chrony configuration on the coreOS and bastion node during installation. This also configure the bastion as a NTP server for the cluster. |
+| chronyconfig.content    | no       | ""             | List of time NTP servers and options pair (see chronyconfig examples). If empty, nodes will still be synced with the bastion node configured as NTP server. |
```

Comment:
"If empty, nodes will still be synced with the bastion node configured as NTP server." This statement is false since we are not doing this in this playbook. Don't you think? Better to mention this in the calller repo ocp4-upi-powervm.

So does it make this variable as required when chronyconfig.enable is true? Atleast a bastion ip should be in the list provided by the user.

Reply:
> This statement is false since we are not doing this in this playbook

right.

> So does it make this variable as required when chronyconfig.enable is true?

If `chronyconfig.enabled=true`and `chronyconfig.content` is not set, this will setup chrony on the bastion node, sync with default ntp server (pool redhat) and also local HW clock (with high stratum)... So I think it is not a required parameter.